projects/schematics/public_api.d.ts

```diff
@@ -0,0 +1 @@
+export * from './src/shared';

```

Comment:
We shouldn't really be committing this file. We don't have it in other libraries. 
Strange that this one didn't ignore it: https://github.com/SAP/spartacus/pull/7554/commits/e2b851a4f5a99ad9726bf638cf4561631de241d3#diff-1f745621faf1fea6435d185420fce42cR4

I guess it's generated once we run `tsc`?